utils/roles/oim_cleanup/oim_container_cleanup/tasks/main.yml

```diff
@@ -49,6 +49,14 @@
     - name: Cleanup iDRAC Telemetry Receiver container
       ansible.builtin.import_tasks: cleanup_idrac_telemetry_receiver.yml
 
+- name: Cleanup prometheus pump and prometheus container
+  when:
+    - hostvars['localhost']['idrac_telemetry_support']
```

Comment:
idrac telemetry check not  required. already within that block. only require prometheus check